.github/workflows/dcm.yaml

```diff
@@ -15,15 +15,19 @@ name: Dart Code Metrics
 
 on:
   pull_request_target
+    branches:
+      - master
 
 jobs:
   flutter-prep:
     uses: ./.github/workflows/flutter-prep.yaml
     with:
       os-name: ubuntu
+      requires-label: 'run-dcm-workflow'
       needs-checkout-merge: true
 
   dcm:
+    if: ${{ contains(github.event.*.labels.*.name, 'run-dcm-workflow') }}
```

Comment:
Is there a reason why we want this on all types of events instead of just for pull_requests?

Reply:
Good point, switched to just pull_requests!